src/Workspaces/Core/Portable/Workspace/Host/PersistentStorage/IPersistentStorageFaultInjectionService.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Host
+{
+    internal interface IPersistentStorageFaultInjector
```

Comment:
❕ Make sure to file a bug to rewrite this using true faults instead of injection (and remove this interface). The current approach is ugly but will get us through the upcoming release.